src/documentmanager/documentmanager.cpp

```diff
@@ -182,6 +184,14 @@ void DocumentManager::loadFile(CodeEditor *newView, const QString &fileName)
 
 void DocumentManager::onSplit(Qt::Orientation orientation)
 {
+    // if there is only one doc area -
```

Comment:
it can be 0 doc area

Reply:
provided additional comment to clarify current block of code